authors/apps/authentication/tests/test_base.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework.test import APITestCase, APIClient
+from django.urls import reverse
+from ..models import User
+
+class TestBase(APITestCase):
+    
+    def setUp(self):
+        self.client = APIClient()
+        self.user_data = {
+                'user': {
+                    'email': "test@testuser.com",
+                    'username': "testuser",
+                    'password': "password"
+                    }
+                }
+        self.test_user = {
+            "user": {
+                "username": "neelxie",
+                "email": "derek@derek.com",
+                "password": "realers"
+            }
+        }
+        self.new_test_user = User.objects.create_user(username="neelxie",
```

Comment:
@neelxie  Good work so far although l noticed you are accessing the create_user method directly from the models l suggest you instead create the user via the endpoint. Just the way you are doing it in the method `test_login_for_signed_up_user`

Reply:
> @neelxie Good work so far although l noticed you are accessing the create_user method directly from the models l suggest you instead create the user via the endpoint. Just the way you are doing it in the method `test_login_for_signed_up_user`

@jennizalwango let me fix exactly that.